### PR TITLE
Add changeset for Primitives + Theming Release

### PR DESCRIPTION
**Pull request by wlee221**
*Issue #, if available:*

*Description of changes:* Adds changeset for primitives + theming release. Added a placeholder for now.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ericclemmons — APPROVED** [review]
I assume we'll make a final pass at the result in #596 ?  Or what's the best place to editorialize & "collapse" them all into our first major release?

(I recognize that subsequent releases are best for changesets to communicate patches/minors)

**Comment by wlee221** [comment]
@ericclemmons Right, the final pass will be at #596. It's just that it's the best practice to add new `changeset` on its own PRs. #596 are just a consumer of changesets (you get to *see* what happens, but not *modify*) and are not meant to be editoralized.

I opened this PR because this "major" bump `changeset` will be at the top of the changelogs and is important enough to be its own PR.

**Review by ericclemmons — CHANGES_REQUESTED** [review]
@dbanksdesign Since this will be top of the changelog, the bulk of it will be editorialized and likely cross-link to docs beyond summarizing what's in this.

File: .changeset/four-melons-double.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/ui-react': major
+'@aws-amplify/ui': major
+---
+
+Added new UI primtives and theming API
```
Comment (ericclemmons) [inline .changeset/four-melons-double.md:6]: @dbanksdesign Can you add some content here that will headline our changelog?

I basically restated the goals & linked the RFC + docs:

https://github.com/aws-amplify/amplify-ui/pull/745/files

**Review by Milan-Shah — APPROVED** [review]

**Review by slaymance — APPROVED** [review]
🚢

**Pull request merged**